package.json

```diff
@@ -42,6 +42,7 @@
     "node-fetch": "^2.6.0",
     "pino": "^5.15.0",
     "prompts": "^2.3.0",
+    "rxjs": "^6.5.4",
```

Comment:
Can we maybe put that as a `devDependency` and do the same stuff as with `node-pty`? Look at the `requireNodePty` function in `e2e-testing.ts`. Looks like the only reason it's a dependency right now is because we want to share that with the plugin

Reply:
Will do!